### ISSUE TITLE
Is OMP_DLL_LOC still needed for Windows?

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#2080]
@joa-quim I saw following codes in the cmake script src/CMakeLists.txt:
https://github.com/GenericMappingTools/gmt/blob/50bcdb885dc30aaf99f043e3d12aeebb814e0a2a/src/CMakeLists.txt#L246-L251

Is OMP_DLL_LOC still necessary for recent cmake releases? If not, I think we can remove these codes.

**Comment by joa-quim** [comment]
It was the last time I checked (not so long ago).

**Comment by seisman** [comment]
The builtin cmake script [InstallRequiredSystemLibraries.cmake](https://github.com/Kitware/CMake/blob/master/Modules/InstallRequiredSystemLibraries.cmake) contains following codes:

```
  # MSVC 8 was the first version with OpenMP
  # Furthermore, there is no debug version of this
  if(CMAKE_INSTALL_OPENMP_LIBRARIES AND _IRSL_HAVE_MSVC)
    set(_MSOMP_DLL_VERSION ${_MSVC_DLL_VERSION})
    set(_MSOMP_IDE_VERSION ${_MSVC_IDE_VERSION})

    if(_MSOMP_DLL_VERSION)
      set(v "${_MSOMP_DLL_VERSION}")
      set(vs "${_MSOMP_IDE_VERSION}")
      set(MSVC_OPENMP_DIR "${MSVC_REDIST_DIR}/${CMAKE_MSVC_ARCH}/Microsoft.${MSVC_REDIST_NAME}.OPENMP")

      if(NOT CMAKE_INSTALL_DEBUG_LIBRARIES_ONLY)
        set(__install__libs ${__install__libs}
          "${MSVC_OPENMP_DIR}/vcomp${v}.dll")
      endif()
    endif()
  endif()
```

I though it will copy the vcomp dll library to the right place. The codes was added before cmake 3.4.0.

**Comment by seisman** [comment]
Closed by #2081.
